### DISCUSSION TITLE
Upgrade from Windows 8.1 Pro to Windows 10/11

### DISCUSSION THREAD

**Discussion by aivanov2-godaddy** [AveYo/MediaCreationTool.bat#104] [Q&A]
I've tried to upgrade from Windows 8.1 Pro to 10 or 11 but I'm receiving the following error:

![image](https://user-images.githubusercontent.com/92979619/153312376-4cafff49-6db2-42e6-a81c-6a5d162239f7.png)

It does not matter what version I've chosse, or where I'm executing the script from

**Comment by AveYo** [comment]
@aivanov2-godaddy: better late than never, but the updated script should generally prevent such non-canonical path issues ;)

**Comment by jdev082** [comment]
This script generally works best when upgrading from a Windows 10 installation, so try upgrading to Windows 10 first?
